app/src/main/java/com/kickstarter/viewmodels/ResetPasswordViewModel.kt

```diff
@@ -127,9 +129,10 @@ interface ResetPasswordViewModel {
         override fun resetError(): Observable<String> {
             return this.resetError
                     .takeUntil(this.resetSuccess)
-                    .map { it.errorMessage() }
+                    //.map { it.errorMessage() } /*  null pointer exception is thrown */
```

Comment:
is probably returning an exception because the filter on line 90 was deleted, bring back the filter and do this
 `.map { it.errorMessage() ?: ""}`
If you do `.map{""}` you will always return an empty string instead of the error envelope message, you want to return the empty string just in case there is not message on the error envelope I guess

Reply:
Question: If we bring back the `.filter()` operator to filter out null emissions, wouldn't it not emit anything at all to the `resetError` stream?

Reply:
@Arkariang Question: If we bring back the .filter(ObjectUtils.isNotNull(it)) operator to filter out null emissions, wouldn't it not emit anything at all to the resetError stream?